Makefile

```diff
@@ -201,19 +201,19 @@ python-black-update: .venv/bin/black
 
 .PHONY: elixir
 elixir: elixir-check-formatted elixir-credo devclean
-	@dev/run -a adm:pass --no-eval 'test/elixir/run --exclude without_quorum_test --exclude with_quorum_test'
+	@dev/run -a adm:pass --no-eval 'test/elixir/run --exclude without_quorum_test --exclude with_quorum_test $(EXUNIT_OPTS)'
 
 .PHONY: elixir-cluster-without-quorum
 elixir-cluster-without-quorum: elixir-check-formatted elixir-credo devclean
 	@dev/run -n 3 -q -a adm:pass \
 	          --degrade-cluster 2 \
-            --no-eval 'test/elixir/run --only without_quorum_test'
+            --no-eval 'test/elixir/run --only without_quorum_test $(EXUNIT_OPTS)'
 
 .PHONY: elixir-cluster-with-quorum
 elixir-cluster-with-quorum: elixir-check-formatted elixir-credo devclean
 	@dev/run -n 3 -q -a adm:pass \
-						--degrade-cluster 1 \
-						--no-eval 'test/elixir/run --only with_quorum_test'
+			--degrade-cluster 1 \
```

Comment:
Please normalise all of the extra indent to exactly 2 tabs, if you would. (lines 209, 215, etc.)